src/components/mdCheckbox/mdCheckbox.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="md-checkbox" :class="[themeClass, classes]">
     <div class="md-checkbox-container" @click.stop="toggleCheck" tabindex="0">
-      <input type="checkbox" :name="name" :id="id" :disabled="disabled" :value="value" :checked="checked" tabindex="-1">
+      <input type="checkbox" :name="name" :id="id || this._uid" :disabled="disabled" :value="value" :checked="checked" tabindex="-1">
```

Comment:
@jdrydn The _uid is an integer that can colide with others IDs in the same page. There is a uniqueId generator in this project. It was used in the tabs component. Could you change this?